cmd/notifier/main.go

```diff
@@ -75,7 +76,8 @@ func main() {
 
 	notifierMetrics := metrics.ConfigureNotifierMetrics(telemetry.Metrics, serviceName)
 	databaseSettings := config.Redis.GetSettings()
-	database := redis.NewDatabase(logger, databaseSettings, redis.Notifier)
+	redis := redis.NewDatabase(logger, databaseSettings, redis.Notifier)
+	database := db.NewDatabaseWithMetrics(redis, "redis", &telemetry.Metrics)
```

Comment:
`"redis"` is not a good prefix for metrics. We have a metrics prefix that is set in config files. Each service adds its own name to prefix. Let's use this as prefix